lib/npm-check-updates.js

```diff
@@ -97,20 +97,54 @@ function createDependencyTable() {
             middle: ''
         }
     });
+    return baseTable;
+}
+
+/**
+ * @param {object} args
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+async function getOwnerPerDependency(args, options) {
+    const packageManager = vm.getPackageManager(options.packageManager);
+    args.owner = {};
+    await Promise.all(
+        Object.keys(args.to).map(async (dep) => {
+            const from = args.from[dep] || null;
+            const to = args.to[dep] || null;
+            const ownerChanged = await packageManager.packageAuthorChanged(dep, from, to);
+            // eslint-disable-next-line require-atomic-updates
+            args.owner[dep] = ownerChanged;
+        })
+    );
+    return args;
 }
 
 /**
  * @param {object} args
  * @param {object} args.from
  * @param {object} args.to
- * @returns {Table}
+ * @param {object} args.owner
+ * @param {object} options
+ * @returns {Promise<Table>}
  */
-function toDependencyTable(args) {
+async function toDependencyTable(args, options) {
     const table = createDependencyTable();
-    const rows = Object.keys(args.to).map(dep => {
-        const from = args.from[dep] || '';
-        const to = versionUtil.colorizeDiff(args.from[dep], args.to[dep] || '');
-        return [dep, from, '→', to];
+    let mappedArgs = [];
+    if (options.ownerChanged) {
+        const newArgs = await getOwnerPerDependency(args, options);
```

Comment:
`toDependencyTable` is simply a formatting function, so this is not an appropriate place to call `getOwnerPerDependency`. Rather, this call should probably be done in `upgradePackageDefinitions` and passed to `printUpgrades`.